src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/submission/submissionApiDelegateImpl/GetPackageInformationTest.java

```diff
@@ -161,6 +160,6 @@ public void withInCorrectIDReturnForbidden() {
                 .mimeType("application/txt")
                 .isSupremeCourtScheduling(null)
                 .isAmendment(null)
-                .data(new JsonObject()).create());
+                .data(new Object()).create());
```

Comment:
are these related to the jaeger refactor? or tech debt found doing the refactor?